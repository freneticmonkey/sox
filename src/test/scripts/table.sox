//TODO:

var t = Table()
t["hello"]

print(t["hello"])
//     <nil>

t["hello"] = "world"

print(t["hello"])
//     world

print(t)

t["after"] = "goodbye"

print(t)

print(t)
//     {"0","hello"}
//     {"after","goodbye"}

// var t2 = Table({
//     "key" : "value",
//     "name": "value"
// })
