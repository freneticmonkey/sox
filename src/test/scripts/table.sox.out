nil
world
<table>{{"hello":"world"}}
<table>{{"after":"goodbye"},{"hello":"world"}}
